CAFAna/Core/Progress.cxx

```diff
@@ -21,7 +21,7 @@ namespace ana
     // If no else is drawing, we can
```

Comment:
This ProgressBar patch is the one we need on the nova side.

Reply:
thanks.  turns out I had this staged, and had just forgot to push it.  done.